server.go

```diff
@@ -3589,14 +3564,16 @@ func HashPassword(password string) ([]byte, error) {
 type ContinuousQuery struct {
 	Query string `json:"query"`
 
-	mu              sync.Mutex
-	cq              *influxql.CreateContinuousQueryStatement
-	lastRun         time.Time
-	intoDB          string
-	intoRP          string
-	intoMeasurement string
+	mu      sync.Mutex
+	cq      *influxql.CreateContinuousQueryStatement
+	lastRun time.Time
 }
 
+func (cq *ContinuousQuery) intoDB() string          { return cq.cq.Source.Target.Measurement.Database }
```

Comment:
I guess `into` comes from the CQ statement. Perhaps `dest` would be clearer to the next guy. Not sure.